src/main/java/com/checkmarx/sdk/config/PropertyFileLoader.java

```diff
@@ -22,11 +22,11 @@
      *
      * @param filenames list of resource filenames to load properties from.
      *                  If the same property appears several times in the files, the property value from a file will be overridden with the value from the next file.
-     * @throws ASTRuntimeException if no filenames is provided, or if an error occurred while loading a file.
+     * @throws ScannerRuntimeException if no filenames is provided, or if an error occurred while loading a file.
      */
     public PropertyFileLoader(String... filenames) {
```

Comment:
Looks like the whole PropertyFileLoader class can be deleted (I see it's no longer used).